src/api/resource/keyable-type/product/category-asset.ts

```diff
@@ -0,0 +1,51 @@
+import { Category } from "@commercetools/platform-sdk";
+import { splitArray } from "../../../../utils/split-actions.js";
+import { existWithoutKeyPredicate } from "../../predicate.js";
+
+export const categoryAssetsQuery = `query CategoryAssetsQuery($predicate: String!) {
+  categories(where: $predicate, sort: "id asc", limit: 500) {
+    total
+    results {
+      id
+      version
+      assets {
+        id
+        key
+      }
+    }
+  }
+}
+`;
```

Comment:
```suggestion
```

Do we need this here? Because I see it already defined in `src/api/resource/keyable-type/category/category-asset.ts`

Reply:
You're right, this was a lift over. I deleted it as suggested